### PR TITLE
ADD CONTRIBUTING_GUIDELINES FILE TO READMEKRAFT

### PR DESCRIPTION
**Pull request by Code-Hacker26**
Here by  a fair attempt is made to add a contribution file dictating the process and guidelines for any contribution in the given repo "READMEKRAFT".

### CODE REVIEW

File: CONTRIBUTING_GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+### Prerequisites
+
+Make sure you have the following installed on your machine:
+
+- Node.js (v14.x or later)
+- npm, yarn, pnpm, or bun (whichever you prefer)
+
+### Setting Up the Project
+
```
Comment (TheShiveshNetwork) [inline CONTRIBUTING_GUIDELINES.md:22]: The first step should be to *fork the repository* then all the steps. Shift the order of these to add this.

File: CONTRIBUTING_GUIDELINES.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    If you have any questions or need assistance, please don't hesitate to reach out. 
+    
+    You can comment directly on the relevant issue, and one of the maintainers or community members will get back to you as soon as possible.
+
+    For general inquiries, feel free to open a new issue with your question. 
+    We're here to help and ensure your contribution experience is smooth .
+
+
+## License ##
+    By contributing to examples, you agree that your contributions will be licensed under the LICENSE file in the root directory of this source tree.
```
Comment (TheShiveshNetwork) [inline CONTRIBUTING_GUIDELINES.md]: Also, add the link to the [LICENSE](https://github.com/TheShiveshNetwork/readmekraft/blob/main/LICENSE) file

**Comment by TheShiveshNetwork** [comment]
Hey @Code-Hacker26 I have added some comments. Resolve them. 
It looks good, also I'll have one request.
Can you make a standard README too? Use our tool for creating [Project Readme](https://readmekraft.vercel.app/createProjectReadme) 
Also link the CONTRIBUTING_GUIDELINES.md file here.

**Comment by Code-Hacker26** [comment]
@TheShiveshNetwork , does Contributing.md need any change or is it good ?

And sure I can look into making README.md file for the same project

**Comment by TheShiveshNetwork** [comment]
> @TheShiveshNetwork , does Contributing.md need any change or is it good ?
> 
> And sure I can look into making README.md file for the same project

I have added a few comments above. Resolve them first, then it will be good to go.

**Comment by Code-Hacker26** [comment]
Hey @TheShiveshNetwork,

As a newcomer to open source, I'm still learning the ropes. I am unable  to find  your comments regarding the changes needed in the CONTRIBUTING.md file. It would be really helpful if we could discuss the details here.

**Comment by TheShiveshNetwork** [comment]
> Hey @TheShiveshNetwork,
> 
> As a newcomer to open source, I'm still learning the ropes. I am unable to find your comments regarding the changes needed in the CONTRIBUTING.md file. It would be really helpful if we could discuss the details here.

I'll explain it here. There are two updates.
1. [Line 17] The first step should be to fork the repository then all the steps. Shift the order of these to add this.
2. [Line 71] Also, add the link to the LICENSE file

**Comment by TheShiveshNetwork** [comment]
@Code-Hacker26 alsk to see my added comments,  go to the files changed section and there you'll see them on the mentioned lines in the above reply like line 17 and 71. Resolve them.
And also add the README file in this pull request as mentioned. Then it can be merged.

**Comment by TheShiveshNetwork** [comment]
No worries @Code-Hacker26 since we require the guidelines ASAP, I'll take it up from here. Good Work. You take up to create README work. I'll create an issue. First comment there that you are interested then you can start working.

**Review by TheShiveshNetwork — APPROVED** [review]

**Review by TheShiveshNetwork — APPROVED** [review]

File: CONTRIBUTING_GUIDELINES.md (hunk 1 above)
Comment (Code-Hacker26) [reply CONTRIBUTING_GUIDELINES.md:22]: @TheShiveshNetwork Thank you for the explanation! I appreciate your help. 
 I'll start working on the README.md .

**Pull request merged**